helm/chaos-mesh/README.md

```diff
@@ -0,0 +1,138 @@
+# Chaos Mesh
+
+[Chaos Mesh](https://github.com/pingcap/chaos-mesh) is a cloud-native Chaos Engineering platform that orchestrates chaos on Kubernetes environments.
+
+## Introduction
+
+This chart bootstraps a [Chaos Mesh](https://github.com/pingcap/chaos-mesh) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Deploy
+
+Before deploying Chaos Mesh, make sure you have installed the [Prerequisites](https://github.com/pingcap/chaos-mesh#prerequisites). And then follow the [deploy-chaos-mesh](https://github.com/pingcap/chaos-mesh#deploy-chaos-mesh) section step by step.
+
+## Parameters
+
+The following tables list the configurable parameters of the Chaos Mesh chart and their default values.
+
+|                 Parameter                  |                                                     Description                                                      |                         Default                         |
+|--------------------------------------------|----------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|
+| `clusterScoped`                            | whether chaos-mesh should manage kubernetes cluster wide chaos.Also see rbac.create and controllerManager.serviceAccount | `true` |
+| `rbac.create` |  | `true`                                                |
+| `enableProfiling` | a flag to enable pprof in controller-manager and chaos-daemon  | `false` |
+| `controllerManager.serviceAccount` | the serviceAccount for chaos-controller-manager | `chaos-controller-manager` |
+| `controllerManager.replicaCount` | replicas for chaos-controller-manager | `1` |
+| `controllerManager.image` | the docker image for chaos-controller-manager  | `pingcap/chaos-mesh:latest` |
+| `controllerManager.imagePullPolicy` | image pull policy | `Always` |
+| `controllerManager.nameOverride` |  |  |
+| `controllerManager.fullnameOverride` |  |  |
+| `controllerManager.service.type` | Kubernetes Service type | `ClusterIP` |
+| `controllerManager.resources` | CPU/Memory resource requests/limits for chaos-controller-manager pod | `requests: { cpu: "250m", memory: "512Mi" }, limits:{ cpu: "500m", memory: "1024Mi" }`   |
+| `controllerManager.nodeSelector` |  Node labels for chaos-controller-manager pod assignment | `{}` |
+| `controllerManager.tolerations` |  Toleration labels for chaos-controller-manager pod assignment | `[]` |
+| `controllerManager.affinity` |  Map of chaos-controller-manager node/pod affinities | `{}` |
+| `controllerManager.podAnnotations` | Deployment chaos-controller-manager annotations | `{}`|
+| `chaosDaemon.image` | the docker image for chaos-daemon | `pingcap/chaos-mesh:latest` |
+| `chaosDaemon.imagePullPolicy` | image pull policy | `Always` |
+| `chaosDaemon.grpcPort` |  | `31767` |
+| `chaosDaemon.httpPort` |  | `31766` |
```

Comment:
Is it better to add some comments about the two ports? Like their usage?

Reply:
Not very familiar with chaosDaemon. I may need some help.
`chaosDaemon.grpcPort` | the port which grpc server listens on 
`chaosDaemon.httpPort`  | the port which http server listens on

code from here
https://github.com/pingcap/chaos-mesh/blob/7ce47e8d6fb9d047fc82a0b555f5b690f42f8942/cmd/chaos-daemon/main.go#L38-L39

It's OK?

Reply:
I think it is okay. Thanks

Reply:
Done!